scripts/minishift-execute-tests.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
```

Comment:
Just being curious, we are going to merge this script with minikube script to test. right?
And we will use this script to setup and run the test for minikube and minishift? 
/cc: @rnapoles-rh

Reply:
yes, done